File: AppClienteVIP/app/src/main/java/app/modelo/appclientevip/SplashActivity.java
package app.modelo.appclientevip;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.os.Handler;

import app.modelo.appclientevip.api.AppUtil;

public class SplashActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_splash);

        telaSplash();
    }

    private void telaSplash() {

        new Handler().postDelayed(() -> {

            Intent intent = new Intent(SplashActivity.this, LoginActivity.class);
            startActivity(intent);
            finish();
            return;

        }, AppUtil.TIME_SPLASH);
    }
}

File: MinhaLojaKotlin/app/src/main/java/app/modelo/minhaloja/SplashActivity.kt
package app.modelo.minhaloja

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.os.Handler
import android.os.Looper

class SplashActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_splash)


        Handler(Looper.myLooper()!!).postDelayed({

        }, 10000)
    }
}